File: Source/Classes/NCCU/TransCell.m
//
//  TransCell.m
//  Mumble
//
//  Created by HanChien Chun on 2016/8/15.
//
//

#import "TransCell.h"

@implementation TransCell

- (void)awakeFromNib {
    [super awakeFromNib];
    // Initialization code
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

- (void)dealloc {
    [_labelUserName release];
    [_imageUser release];
    [_price release];
    [super dealloc];
}
@end

File: Source/Classes/NCCU/Transtable/TransCell.swift
//
//  TransCell.swift
//  Mumble
//
//  Created by William on 2017/4/27.
//
//

import UIKit

class TransCell: UITableViewCell {
    
    @IBOutlet var labelUserName: UILabel!
    @IBOutlet var imageUser: UIImageView!
    @IBOutlet var price: UILabel!
    
    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }
    
    override func setSelected(_ selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)
        // Configure the view for the selected state
    }
    
    deinit {
    }
}

